README.md

```diff
@@ -34,6 +34,13 @@ You can add as many aliases as you want in the corresponding configuration file
 Each alias is a key/value combination:
 > daily_alias = BSDEV-42
 
+### Internal issue references
+
+Internal issue prefixes are used to avoid adding 'to dispatch' time on internal entries
+
+You can add as many internal issue prefixes as you want in the corresponding configuration file section.
+> internal_issue_prefixes = BS-, BSRD-, ABCH-, ABFR-
```

Comment:
why do you need this list? You know already which lines have to be ignored:

* "arrive" or similar
* slack time marked as `**`
* any alias declared in the config

N'est pas?

Reply:
> BTW do we have an RD card for this?

Not yet, it was a side improvement for now ^^ But if we could have one, should be nice